src/components/Visualization/Visualization.js

```diff
@@ -302,6 +303,57 @@ export const Visualization = ({
 
     const isInModal = !!filters?.relativePeriodDate
 
+    const cellIsUndefined = (rowIndex, columnIndex) => {
+        const row = (data?.rowContext || {})[rowIndex]
+        if (row && row[columnIndex]?.valueStatus == NOT_DEFINED_VALUE) {
```

Comment:
Would prefer `===` as a standard unless there's a specific need to do `==` (which typically should be avoided I think).